docs/source/tutorials/basicrestapi.rst

```diff
@@ -111,7 +111,7 @@ Notice what happens if we try to request a city that's not in our
 
     {
         "Code": "ChaliceViewError",
-        "Message": "ChaliceViewError: An internal server error occurred."
```

Comment:
@jamesls This is the only instance I found in the docs that reflect the current behavior. All others interestingly do not include the class name at the beginning :thinking: